backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/descriptors/DescriptorUtils.kt

```diff
@@ -254,6 +263,11 @@ fun IrConstructorCall.getStringValue(name: String): String {
     return (getValueArgument(parameter.index) as IrConst<String>).value
 }
 
+fun <T> IrConstructorCall.getArgumentValueOrNull(name: String): T? {
```

Comment:
I guess the name of this function should clearly indicate that it is applicable to annotations only.